src/api/models/mermaid.py

```diff
@@ -756,6 +757,10 @@ def sample_unit(self):
     def __str__(self):
         return str(_(u'transect method'))
 
+    @property
```

Comment:
Is this needed anymore? I can't see a reference, but maybe it's there.

Reply:
Yeah it gets used in the signals...

```
@receiver(post_delete, sender=TransectMethod)
def update_summaries_on_delete_transect_method(sender, instance, *args, **kwargs):
...
```